.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Build manage-help-content-publisher
+
+on:
+  pull_request:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+
+jobs:
+  CI:
+    permissions:
+      id-token: write
+      contents: read
+    name: manage-help-content-publisher build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Env
+        run: env
+
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-region: eu-west-1
+          role-to-assume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      - name: Setup Java
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'corretto'
+          java-version: '11'
+          cache: 'sbt'
+
+      - name: Compile and package project
+        run: sbt clean assembly
```

Comment:
just check that assembly actually runs the tests, and if they fail, the build doesn't upload to riffraff

Reply:
possibly need this
```suggestion
        run: sbt clean test assembly
```

Reply:
The assembly task runs the tests by default 🙂

Reply:
good that's fine then 👍